linux_os/guide/system/software/system-tools/package_tuned_removed/rule.yml

```diff
@@ -20,6 +20,9 @@ identifiers:
     cce@rhel7: 82905-1
     cce@rhel8: 82904-4
 
+references:
+    srg: SRG-OS-000095-GPOS-00049
```

Comment:
Is tuned installed by default? If not, this mapping and rule should be removed.

Reply:
Yes, it is installed by default.

Reply:
What's the case for it's removal?

Reply:
Good question.  The rule is in the OSPP profile and but I'm not sure why.

Reply:
@stevegrubb can you help with clarification why "tuned" is removed in the OSPP profile?

Reply:
Yes, traditionally, labs ask us to either remove root running daemons that are not essential for system operation or have them to use capabilities and run under another account. Tuned does not run under another account so we remove it as a hardening step.

Reply:
Thanks @stevegrubb. 

Given tuned is installed by default, runs as root, and cannot be configured to run non-privileged, not sure there is any option but removal. Frustrating since tuned would be a UX improvement for DoD system operators. 

Opened https://github.com/ComplianceAsCode/content/issues/4995 to track adding "allowed per mission need" to XCCDF description tags. 

Ack to this mapping.